src/main/java/com/mineboundteam/minebound/crafting/ArmorForgeRecipe.java

```diff
@@ -0,0 +1,72 @@
+package com.mineboundteam.minebound.crafting;
+
+import com.mineboundteam.minebound.registry.RecipeRegistry;
+import net.minecraft.core.NonNullList;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.Container;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.Ingredient;
+import net.minecraft.world.item.crafting.Recipe;
+import net.minecraft.world.item.crafting.RecipeSerializer;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraft.world.level.Level;
+import org.jetbrains.annotations.NotNull;
+
+public class ArmorForgeRecipe implements Recipe<Container> {
+    private final NonNullList<Ingredient> ingredients;
+    private final ResourceLocation resourceLocation;
+    private final ItemStack output;
+
+    public ArmorForgeRecipe(ResourceLocation resourceLocation, NonNullList<Ingredient> ingredients, ItemStack output) {
```

Comment:
Please add an additional ingredient argument for the armor tier necessary for crafting